docs/content/get-started/policies/policies.md

```diff
@@ -33,7 +33,7 @@ first.
 
 Advanced users can learn about designing new policies by following the example
 circuit created in
-[detecting overload](/applying-policies/alerting/detecting-overload.md)
```

Comment:
This seems the correct format so is there any reason for changing it with relative path?

Reply:
it seems to be throwing on local, not it's not showing, I just checked. I don't know that behavior seems ambiguous.

Reply:
It might be possible that the local symlink is not updated. Has happened with me in past and re-creating the symlink fixes the missing new files

Reply:
reverting the change of this url.